src/routes/user/userparentalcontrol.tsx

```diff
@@ -223,7 +223,8 @@ const UserParentalControl: FunctionComponent = () => {
                 throw new Error('Unexpected null user.Policy');
             }
 
-            user.Policy.MaxParentalRating = parseInt((page.querySelector('#selectMaxParentalRating') as HTMLSelectElement).value || '0', 10) || null;
+            const parentalRating = parseInt((page.querySelector('#selectMaxParentalRating') as HTMLSelectElement).value || '0', 10);
+            user.Policy.MaxParentalRating = parentalRating !== undefined ? parentalRating : null;
```

Comment:
parseInt will return a number or NaN so checking for undefined shouldn't do anything

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#return_value

Reply:
So we need to check for NaN?

Reply:
yeah, and because it's JavaScript you need to use the isNaN function, don't do `== NaN` because that's always false